wp-build-hooks.php

```diff
@@ -17,12 +17,23 @@
 	'gatsby'    => 'Gatsby Cloud',
 	'netlify'   => 'Netlify',
 ];
+const BUILD_HOOK_DOMAINS                 = [
+	'pantheon.io' => [
```

Comment:
I think this should be `pantheonsite.io`

Reply:
🤦 my bad